front_end/src/utils/charts.ts

```diff
@@ -579,8 +579,20 @@ export function generateChoiceItemsFromGroupQuestions(
 
   return choiceOrdering.map((order, index) => {
     const question = questions[order];
-    const history = question.aggregations.recency_weighted.history;
     const label = question.label;
+    const closeTime = Math.min(
+      new Date(question.scheduled_close_time).getTime(),
+      new Date(
+        question.actual_resolve_time ?? question.scheduled_resolve_time
+      ).getTime()
+    );
```

Comment:
You can use `question.status` for such cases. This field is generated on the backend and returns 4 primitive statuses: `upcoming|resolved|closed|open`

BE reference: https://github.com/Metaculus/metaculus/blob/68c5031ebbfcf955a4f540b46fd030af68959092/questions/models.py#L142

Reply:
Not sure what you mean, @hlbmtc . I don't need the status (i.e.: whether the q is closed or not), but I need the timestamp when it was closed.